pkg/simulator/terraform_test.go

```diff
@@ -1,49 +1,86 @@
 package simulator_test
 
 import (
-	"github.com/controlplaneio/simulator-standalone/pkg/simulator"
+	sim "github.com/controlplaneio/simulator-standalone/pkg/simulator"
 	"github.com/stretchr/testify/assert"
 	"go.uber.org/zap"
-
 	"testing"
+
+	"os"
 )
 
+var pwd, _ = os.Getwd()
+var tfVarsDirAbsolutePath = pwd + "/" + fixture("noop-tf-dir")
 var noopLogger = zap.NewNop().Sugar()
 
 var tfCommandArgumentsTests = []struct {
-	prepArgs   []string
+	prepArgs  []string
 	arguments []string
 }{
-	{[]string{"output", "test-bucket"}, []string{"output", "-json"}},
-	{[]string{"init", "test-bucket"}, []string{"init", "-input=false", "--var-file=settings/bastion.tfvars", "-backend-config=bucket=test-bucket"}},
-	{[]string{"plan", "test-bucket"}, []string{"plan", "-input=false", "--var-file=settings/bastion.tfvars"}},
-	{[]string{"apply", "test-bucket"}, []string{"apply", "-input=false", "--var-file=settings/bastion.tfvars", "-auto-approve"}},
-	{[]string{"destroy", "test-bucket"}, []string{"destroy", "-input=false", "--var-file=settings/bastion.tfvars", "-auto-approve"}},
+	{[]string{"output"}, []string{"output", "-json"}},
+	{[]string{"init"}, []string{"init", "-input=false", "--var-file=" + tfVarsDirAbsolutePath + "/settings/bastion.tfvars", "-backend-config=bucket=test-bucket"}},
+	{[]string{"plan"}, []string{"plan", "-input=false", "--var-file=" + tfVarsDirAbsolutePath + "/settings/bastion.tfvars"}},
+	{[]string{"apply"}, []string{"apply", "-input=false", "--var-file=" + tfVarsDirAbsolutePath + "/settings/bastion.tfvars", "-auto-approve"}},
+	{[]string{"destroy"}, []string{"destroy", "-input=false", "--var-file=" + tfVarsDirAbsolutePath + "/settings/bastion.tfvars", "-auto-approve"}},
```

Comment:
If you "pull up" the  `"--var-file=" + tfVarsDirAbsolutePath + "/settings/bastion.tfvars"` into a variable above called eg. `testVarFileArg` it will make this block easier to read